test/tst_dap.py

```diff
@@ -7,8 +7,8 @@
 # test accessing data over http with opendap.
 
 yesterday = datetime.utcnow() - timedelta(days=1)
-URL = "http://nomads.ncep.noaa.gov/dods/gfs_1p00/gfs%s/gfs_1p00_00z" % yesterday.strftime('%Y%m%d')
-URL_https = 'https://podaac-opendap.jpl.nasa.gov/opendap/allData/modis/L3/aqua/11um/v2019.0/4km/daily/2017/365/AQUA_MODIS.20171231.L3m.DAY.NSST.sst.4km.nc'
+URL = f'http://nomads.ncep.noaa.gov/dods/gfs_1p00/gfs{yesterday:%Y%m%d}/gfs_1p00_00z'
```

Comment:
@jswhit this URL is being redirect to HTTPS so, in a way, there is not HTTP test and the second one is redundant.
I changed it to a more stable URL anyway b/c the open podaac server is gone and it requires a registration now.